test/utils/successfulResponse.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiResponseOnSuccess } from '../../src/utils/response/apiResponse';
```

Comment:
I don't like this file very much, I think it hides some information that we want to have visible when writing tests.

Reply:
Removed